### DISCUSSION TITLE
When trying to upload a file through the web interface, getting "Error closing file "<filename>": failure"

### DISCUSSION THREAD

**Discussion by GiladHi** [drakkan/sftpgo#1292] [General]
Hi,

I've just installed SFTPGO and created a test user.

When I'm trying to upload a file through the web interface, I'm getting the following error:
"Error closing file "<filename": failure".
I don't think it's a permission issue...

Any ideas?

**Comment by drakkan** [comment]
Hello, please read or share the full sftpgo logs. They should contain more details. Thank you

**Reply by GiladHi** [reply]
Thanks, I had to configure the logs with the ENV file. I'll do more tests and report back.
